src/components/Button.js

```diff
@@ -0,0 +1,10 @@
+export default function Button({ onClick, tailwind, text }) {
+  return (
+    <button
+      onClick={onClick}
+      className={`w-full bg-orange-yellow text-midnight-green rounded p-2 text-lg font-medium hover:bg-caribbean-green ${tailwind}`}
+    >
+      {text}
+    </button>
+  );
+}
```

Comment:
What does `tailwind` variable do for the component and how is it meant to be used?

Reply:
@yenly thanks for asking that! I forgot to talk about that up above. 

If you need to specify a utility class for your button that it doesn't yet encompass, something like a margin top `mt-5`, you can pass in another utility class as `tailwind` and "customize" the component a bit. 

It's not perfect because you can't always OVER-RIDE a style, but you can add a utility class.

Reply:
Thanks for explaining this. It would be helpful to see code comments on why and how to use this `tailwind` variable. Like what kind of customization is allowed. Is it just for spacing? or can I pass my own custom CSS class like the following?
```
.yenStyles {
  background-color: hotpink;
  border: 3px dotted limegreen;
  padding-right: 180px;
}
```

Reply:
I wrote my reply before first, but I think changing `tailwind` to something like `additionalTailwindUtilityClasses` or something similar will help clarify. But I am also happy to put in a comment as well!